src/loaders/common/loader_helper.py

```diff
@@ -586,3 +586,26 @@ def dump_json_to_file(json_file_path: str | Path, json_data: dict[str, Any] | li
     """
     with open(json_file_path, "w", encoding="utf8") as json_file:
         json.dump(json_data, json_file, indent=2)
+
+
+def create_meta_file(
+        output_dir: Path | str,
+        upa_format: str,
```

Comment:
Maybe just `upa`?

Reply:
For many KBasers, the initial impression of a UPA is in the format of `a/b/c` as a workspace object reference.  However, for collection, UPAs are typically structured as 'a_b_c' to align with the file structure. That's why I named `upa` in formate of `a_b_c` differently.

Reply:
I updated it to `structured_upa`. Maybe that's a less confusing name.